src/babel.js

```diff
@@ -13,10 +12,136 @@ function joinExpressionsWithSpaces (expressions, t) {
     }
     quasis.push(t.templateElement({ cooked: ' ', raw: ' ' }, true))
   })
-  return t.templateLiteral(
-    quasis,
-    expressions
-  )
+  return t.templateLiteral(quasis, expressions)
+}
+
+function parseStyledDynamicValues (rules, t, composes = 0, vars) {
```

Comment:
My only concern here is maintainability. I'm not sure if we will need to support more than `attr` in the future.